docs/config/README.md

```diff
@@ -85,7 +85,7 @@ In the second part, which is enclosed in a `()`, is a [style string](#style-stri
 For example:
 
 - `[on](red bold)` will print a string `on` with bold text colored red.
-- `[⬢ $version](bold green)` will print a symbol `⬢ ` followed by the content of variable `version`, with bold text colored green.
+- `[⌘ $version](bold green)` will print a symbol `⌘ ` followed by the content of variable `version`, with bold text colored green.
```

Comment:
@whi-tw Why did you pick ⌘ as a replacement instead of going with _new_ NodeJS symbol?

Reply:
The new symbol doesn't render in the browser (or in a text editor if using an unpatched font)

Using a symbol that's included in most (all?) Fonts, it ensures that this part of the documentation is readable / understandable for anyone - especially as that section of the docs isn't nodejs-specific.

I thought I'd put this reasoning in the commit message, but it looks like I didn't force push my most recent message amendment before merging, so it still (confusingly) refers to the old symbol.

Reply:
Ah right, that makes sense! Thanks for the explanation :+1: